src/state/store/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, compose } from "redux";
+import rootReducer from "../reducers/rootReducer";
+
+const configureStore = () => {
+  return createStore(
+    rootReducer,
+    compose(
+      window.__REDUX_DEVTOOLS_EXTENSION__ &&
```

Comment:
why is it needed?
I dont understand, hilfe!

Reply:
This is for enabling Redux dev tools! Real-time redux state monitoring in your browser 😎

Reply:
deleted :(